src/modules/facility/facility.interface.ts

```diff
@@ -0,0 +1,35 @@
+import { DateOnlyString } from '@ukef/helpers';
+
+export interface Facility {
+  dealIdentifier: string;
+  facilityIdentifier: string;
+  portfolioIdentifier: string;
+  dealBorrowerIdentifier: string;
+  maximumLiability: number;
+  productTypeId: string;
+  capitalConversionFactorCode: string;
+  currency: string;
+  guaranteeCommencementDate: DateOnlyString;
+  guaranteeExpiryDate: DateOnlyString;
+  nextQuarterEndDate: DateOnlyString;
+  facilityInitialStatus: string;
+  facilityOverallStatus: string;
+  delegationType: string;
+  intrestOrFeeRate: number;
+  facilityStageCode: string;
+  exposurePeriod: string;
+  creditRatingCode: string;
+  guaranteePercentage: number;
+  premiumFrequencyCode: string;
+  riskCountryCode: string;
+  riskStatusCode: string;
+  effectiveDate: DateOnlyString;
+  foreCastPercentage: number;
+  issueDate: DateOnlyString;
```

Comment:
This can be null, if facility is not issued. Example 0030598946

Reply:
Good point, I've changed to `DateOnlyString | null`.